src/ucp/core/ucp_mm.c

```diff
@@ -106,6 +106,26 @@ static ucs_status_t ucp_memh_reg_mds(ucp_context_h context, ucp_mem_h memh,
     return UCS_OK;
 }
 
+ucs_status_t ucp_memory_type_detect_mds(ucp_context_h context, void *addr, size_t length,
+                                        uct_memory_type_t *mem_type)
+{
+    unsigned i;
+
+    *mem_type = UCT_MD_MEM_TYPE_HOST;
```

Comment:
this assignment can be moved to line 126, just before returning

Reply:
fixed